tests/test_loose_version.py

```diff
@@ -0,0 +1,71 @@
+#
+# MIT License
+#
+# (C) Copyright 2019-2024 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+Tests for the XName utility class.
+"""
+
+import unittest
+
+from sat.loose_version import LooseVersion
+
+
+class TestLooseVersion(unittest.TestCase):
+    """Tests for the XName class"""
+
+    def setUp(self):
+        self.examples = [
+            "1.0.0",
+            "1.0.0",
+            "1.0.2",
+            "2.1.3",
+            "20240129105721.3d243a4b5120",
+            "1.9.16-20240522115029.beaa1ce7a544",
+            "1.9.18-20240625143747.59e8b16343aa",
+            "2.1.1-64-cos-3.0-aarch64",
+            "23.10.30-20231027",
+            "2.2.0-57-cos-3.0-x86-64"
+        ]
+
+    def test_equality(self):
+        """Test equality of loose versions"""
```

Comment:
I think this test is actually just testing the `version_str` attribute of the `LooseVersion` class, which is a fine test to have, but I'd update the method name and docstring accordingly. Something like:

```suggestion
    def test_version_str(self):
        """Test version_str property stores unmodified version string"""
```